TypeCobol.Test/Parser/Programs/Cobol85/ContinuationLine/ContinuationLine.rdzMix.txt

```diff
@@ -0,0 +1,72 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. DVZF0OSM.
+       data division.
+       working-storage section.
+       01 syntax-ok PIC X(200) VALUE "This text is way too long to fit o
+      -                        "n a single line !".
+       01 var1 pic x(100) value "hi".
+       PROCEDURE DIVISION.
+             move 'CHAMPS 082 A 085 INTERDITS EN OUVERTURE'
+Line 10[19,21] <27, Error, Syntax> - Syntax error : Symbol foo is not referenced
+      -        to foo
+      
+
+      *    OK
+           display "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+      -            " er zer "
+           display 'endxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx1234xxxxxxx
+      -            ' gggg '
+           display 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
Can you add the same display statement but with delimiter `"` ?

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1604/commits/80c7f30d19923763d5f431814c12d34e64324802